schemas/alpha/azure.yaml.json

```diff
@@ -546,6 +546,11 @@
                     "title": "The tag that will be applied to the built container image.",
                     "description": "If omitted, a unique tag will be generated based on the format: {appName}/{serviceName}-{environmentName}:azd-deploy-{unix time (seconds)}. Supports environment variable substitution. For example, to generate unique tags for a given release: myapp/myimage:${DOCKER_IMAGE_TAG}"
                 },
+                "registry": {
+                    "type": "string",
```

Comment:
Can we define a new type for the schema like `expandableString` ?    As a customer, I would like to know what fields I can use expansion or not.  Using the description seems a fragile approach.

Reply:
The goal is for it to look and act like a string and not expose any additional properties and options.  I will look into it for a future PR since there are numerous config options at this point that us `ExpandableString`.